sale_commission_formula/README.rst

```diff
@@ -14,16 +14,16 @@ Sale Commission Formula
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fcommission-lightgray.png?logo=github
-    :target: https://github.com/OCA/commission/tree/12.0/sale_commission_formula
+    :target: https://github.com/OCA/commission/tree/13.0/sale_commission_formula
     :alt: OCA/commission
 .. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
-    :target: https://translation.odoo-community.org/projects/commission-12-0/commission-12-0-sale_commission_formula
+    :target: https://translation.odoo-community.org/projects/commission-13-0/commission-13-0-sale_commission_formula
     :alt: Translate me on Weblate
 .. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
-    :target: https://runbot.odoo-community.org/runbot/165/12.0
+    :target: https://runbot.odoo-community.org/runbot/165/13.0
     :alt: Try me on Runbot
 
-|badge1| |badge2| |badge3| |badge4| |badge5| 
+|badge1| |badge2| |badge3| |badge4| |badge5|
```

Comment:
It seems you have edited this by hand, isn't it?

Reply:
Done